integration-tests/cqlserver/client.go

```diff
@@ -22,6 +24,7 @@ func NewCqlClient(addr string, port int, username string, password string, conne
 	}
 	proxyAddr := fmt.Sprintf("%s:%d", addr, port)
 	clt := client.NewCqlClient(proxyAddr, authCreds)
+	clt.ReadTimeout = time.Second * 600
```

Comment:
Can you comment why this was needed?

Reply:
oh I think this was for debug purposes oops good catch